packages/deployments/contracts/deploy/01_deployMessaging.ts

```diff
@@ -286,8 +287,9 @@ const handleDeploySpoke = async (
   const merkleContract = await hre.ethers.getContractAt("MerkleTreeManager", merkleTreeManager.address, deployer);
   console.log("merkleContract: ", merkleContract.address);
 
-  console.log("await merkleContract.arborist(): ", await merkleContract.arborist());
-  if (!(await merkleContract.arborist())) {
+  const arborist = await merkleContract.arborist();
+  console.log("arborst for spoke merkle ", arborist);
+  if (arborist == constants.AddressZero) {
```

Comment:
we should remove this to ensure configuration only happens in the `initialize` scripts and not the `deploy` ones

Reply:
removed